packages/angular/src/schematics/init/init.spec.ts

```diff
@@ -34,16 +38,18 @@ describe('init', () => {
   });
 
   it('should add a postinstall script for ngcc', async () => {
-    const tree = await runSchematic(
-      'init',
-      {
-        unitTestRunner: 'karma',
-      },
-      appTree
-    );
+    // ARRANGE
+    const host = createTreeWithEmptyWorkspace();
+    await init(host, {
+      unitTestRunner: UnitTestRunner.Karma,
+      linter: Linter.EsLint,
+      skipFormat: false,
+    });
 
-    const packageJson = readJsonInTree(tree, 'package.json');
+    // ACT
+    const packageJson = readJson(host, 'package.json');
```

Comment:
Not that it makes any difference, they are just comments and it's probably a bad copy 🙂  but the `ARRANGE, ACT, ASSERT` are misplaced here and in other test cases below. The `ACT` should be the execution of the generator.

Reply:
I'll go back through and fix them up! :)